src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs

```diff
@@ -377,86 +383,84 @@ private void InnerStartOperationReceiveMessageFrom()
             // The sockaddr is pinned with a GCHandle to avoid complicating the use of Overlapped.
             // WSAMsg contains a pointer to a WSABuffer array describing data buffers.
             // WSAMsg also contains a single WSABuffer describing a control buffer.
-            // 
             PinSocketAddressBuffer();
 
             // Create and pin a WSAMessageBuffer if none already.
-            if (_WSAMessageBuffer == null)
+            if (_wsaMessageBuffer == null)
             {
-                _WSAMessageBuffer = new byte[s_WSAMsgSize];
-                _WSAMessageBufferGCHandle = GCHandle.Alloc(_WSAMessageBuffer, GCHandleType.Pinned);
-                m_PtrWSAMessageBuffer = Marshal.UnsafeAddrOfPinnedArrayElement(_WSAMessageBuffer, 0);
+                _wsaMessageBuffer = new byte[s_wsaMsgSize];
+                _wsaMessageBufferGCHandle = GCHandle.Alloc(_wsaMessageBuffer, GCHandleType.Pinned);
+                _ptrWSAMessageBuffer = Marshal.UnsafeAddrOfPinnedArrayElement(_wsaMessageBuffer, 0);
             }
 
             // Create and pin an appropriately sized control buffer if none already
-            IPAddress ipAddress = (m_SocketAddress.Family == AddressFamily.InterNetworkV6
-                ? m_SocketAddress.GetIPAddress() : null);
-            bool ipv4 = (_currentSocket.AddressFamily == AddressFamily.InterNetwork
-                || (ipAddress != null && ipAddress.IsIPv4MappedToIPv6)); // DualMode
+            IPAddress ipAddress = (_socketAddress.Family == AddressFamily.InterNetworkV6 ? _socketAddress.GetIPAddress() : null);
+            bool ipv4 = (_currentSocket.AddressFamily == AddressFamily.InterNetwork || (ipAddress != null && ipAddress.IsIPv4MappedToIPv6)); // DualMode
```

Comment:
I know that we're generally not concerned about line width but this was already <120 chars width. Why expanding these calls on a single line, making it harder to read?